tests/integrationv2/Makefile

```diff
@@ -71,9 +71,11 @@ test_signature_algorithms:
 	$(call run_tox,$@.py)
 test_version_negotiation:
 	$(call run_tox,$@.py)
+test_cross_compatibility:
```

Comment:
does this not work with cmake then? is there a task/issue to also make it accessible from cmake?

Reply:
Yeah, this honestly shouldn't even be in a makefile since it's just running bash commands IMO. So yes, this wouldn't work with cmake. Moving everything to cmake is another task that eventually needs to be done. Created new issue: #3152